src/components/FieldBuilderTable/FieldBuilderTable.tsx

```diff
@@ -38,8 +47,13 @@ const FieldBuilderTable: FC<FieldBuilderTableProps<FormData>> = ({
       <Tbody>
         {fieldRows.map((fieldRow, index) => (
           <Tr key={fieldRow.id}>
-            {fieldRow.inputs.map((fieldInput) => (
-              <Td key={fieldInput.key}>{fieldInput}</Td>
+            {fieldRow.inputs.map((fieldInput, fieldInputIndex) => (
+              <Td
+                key={fieldInput.key}
+                {...(fieldInputIndex === 0 && { className: 'pf-v5-u-pl-0' })}
```

Comment:
can you add classname pkg it can handle those situations

Reply:
Sure. I noticed it wasn't a dependency so I avoided it thinking there was a reason for that, but I can certainly add that.